easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.8.4-iccifort-2015.2.164-GCC-4.9.2.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '1.8.4'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'iccifort', 'version': '2015.2.164-GCC-4.9.2'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-openib '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--with-hwloc=$EBROOTHWLOC '  # hwloc support
+configopts += ' CC=icc FC=ifort CXX=icpc '
```

Comment:
@pescobar: why did you hardcode `CC` and co here? Any particular reason, because it works fine without this as far as I can tell... Asking because of #2472